src/pages/Setup/FinalDialog.tsx

```diff
@@ -47,7 +49,7 @@ const FinalDialog: FC<Props> = ({ setupPhase, seedWords, callback }) => {
 
           {!!seedWords && (
             <ol className="flex h-[26rem] w-full list-decimal flex-col flex-wrap gap-x-8 rounded-3xl bg-tertiary pl-20 pt-3 font-bold lowercase">
-              {words.map((word, i) => {
+              {seedWords.split(", ").map((word, i) => {
```

Comment:
Will this throw if seedWords is `null`?

Reply:
No, seedwords ist checked 3 lines before :)